src/oncall/ui/static/js/oncall.js

```diff
@@ -558,6 +594,31 @@ var oncall = {
                 }
               }
             }
+          },
+          {
+            name: 'users',
+            hint: true,
+            async: true,
+            highlight: true,
+            limit: typeaheadLimit,
+            displayKey: 'name',
+            source: users,
+            templates: {
+              header: function(){
+                return '<h4> Users </h4>';
+              },
+              suggestion: function(resp){
```

Comment:
CSS is broken when there is a user match, but no service. Add a horizontal divider above the Users suggestion box.